docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,286 @@
+  version/commit in which the problem doesn't happen?
+* **Can you reliably reproduce the issue?** If not, provide details
+  about how often the problem happens and under which conditions it
+  normally happens.
+* **Can you reproduce the problem on another machine?**
+
+Include details about your configuration and environment:
+
+* **Add as an attachment** (or add the contents of) the following:
+  - The text output by SpECTRE (including any stack trace)
+  - The input file(s)
+  - $SPECTRE_BUILD_DIR/LibraryVersions.txt
+* **What is the name and version of the OS you're using**?
+* If possible (for SXS computers or HPC systems), a **path to a run
+  directory** that is accessible by SpECTRE core developers.
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion
+for SpECTRE, including completely new features and minor improvements
+to existing functionality. Following these guidelines helps
+maintainers and the community understand your suggestion and find
+related suggestions.
+
+Before creating enhancement suggestions, please **perform a
+[search](https://github.com/sxs-collaboration/spectre/issues)** to see
+if the enhancement has already been suggested. If it has, add a
+comment to the existing issue instead of opening a new one.
+
+#### How Do I Submit A (Good) Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub
+issues](https://guides.github.com/features/issues/).  When you are
+creating an enhancement suggestion, please **include as many details
+as possible** as you fill in the template.
+
+* **Use a clear and descriptive title** for the issue to identify the
+  suggestion.
+* **Provide a step-by-step description of the suggested enhancement**
+  in as many details as possible.
+* **Explain why this enhancement would be useful** to most SpECTRE users.
+
+### Your First Code Contribution
+
+Unsure where to begin contributing to SpECTRE? You can start by
+looking through these `good first issue` and `help wanted` issues:
+
+* [Good first issues][good-first-issue] - issues which should only
+  require a few lines of code, and a test or two.
+* [Help wanted issues][help-wanted] - issues which should be a bit
+  more involved than `good first issue`s.
+
+#### Local development
+
+SpECTRE can be developed locally. For instructions on how to do this,
+see the following sections in the [SpECTRE
+documentation](https://spectre-code.org/):
+
+* [Installing SpECTRE](installation.html)
+* [Running Status Checks Locally](travis_guide.html)
+
+### Pull Requests
+
+The process described here has several goals:
+
+- Maintain SpECTRE's quality
+- Fix problems that are important to users
+- Engage the community in working toward the best possible code
+- Enable a sustainable system for SpECTRE's maintainers to review
+  contributions
+
+Please follow these steps to have your contribution considered by the
+maintainers:
+
+1. Follow all instructions in the template.
```

Comment:
The checkboxes are ambiguous to me. Am I supposed to check them off myself when submitting, or do reviewers do this? Some can't be checked until Travis finishes, e.g."

Reply:
I agree, the check boxes are ambiguous. Remove them?

Reply:
There are no checkboxes in this PR; please file an issue if you want to discuss the already existing template.